locales/ca/LC_MESSAGES/volto.po

```diff
@@ -2303,8 +2303,8 @@ msgid "Ok"
 msgstr "D'acord"
 
 #: components/manage/Widgets/IdWidget
-# defaultMessage: Only 7-bit bytes characters are allowed. Cannot contain characters like:"<", ">", "&", "#", "/", "?". Cannot start with: "_", "aq_", "@@", "++". Cannot end with "__". Cannot be ".", "..", double quotes(can be traversed). Cannot be "REQUEST" or any characters that are illegal in URLs. No newlines.
-msgid "Only 7-bit bytes characters are allowed. Cannot contain characters like:"<", ">", "&", "#", "/", "?". Cannot start with: "_", "aq_", "@@", "++". Cannot end with "__". Cannot be ".", "..", double quotes(can be traversed). Cannot be "REQUEST" or any characters that are illegal in URLs. No newlines."
+# defaultMessage: Only 7-bit bytes characters are allowed. Cannot contain characters like:<, >, &, #, /, ?. Cannot start with: _, aq_, @@, ++. Cannot end with __. Cannot be these specific keywords: REQUEST, ., ..,"", or any other characters that are illegal in URLs. No new lines allowed.
```

Comment:
Sorry for my incomplete first pass. This time I am being thorough.

1. Should this keyword `""` be `"`, or is it intended to be empty string?
2. Similarly is `..` really a keyword?
3. Let's combine the two separate parts that define illegal characters into one sentence. This should be done everywhere, and retranslated.
4. Also I made the final sentence wording to be consistent with the others.
5. I corrected some separation and spacing of character lists.

```suggestion
# defaultMessage: Only 7-bit bytes characters are allowed. Cannot contain characters like: <, >, &, #, /, ?, or others that are illegal in URLs. Cannot start with: _, aq_, @@, ++. Cannot end with __. Cannot be these specific keywords: REQUEST, ., .., "". Cannot contain new lines.
```

FWIW, I think it is *much* easier to read without quoting all the entities. Thanks for doing this!

Reply:
Thank you also for the suggestions! I will update the text today. Regarding the `""`, if you write nothing in the input, it is still valid. Must be invalidated when you type specifically `""` like a single word. At least that's what I understood from the issue.